docker-compose.yaml

```diff
@@ -19,37 +20,36 @@ services:
       - argilla
 
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.11.1
-    container_name: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.5.0
+    ports:
+      - "9200:9200"
+      - "9300:9300"
     environment:
       - node.name=elasticsearch
-      - cluster.name=es-local
+      - cluster.name=es-argilla-local
       - discovery.type=single-node
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
       - cluster.routing.allocation.disk.threshold_enabled=false
+      - xpack.security.enabled=false
     ulimits:
       memlock:
         soft: -1
         hard: -1
     networks:
       - argilla
     volumes:
-      - elasticdata:/usr/share/elasticsearch/data
-
+      - elasticdata85:/usr/share/elasticsearch/data/
   kibana:
-    image: docker.elastic.co/kibana/kibana:7.11.1
-    container_name: kibana
+    image: docker.elastic.co/kibana/kibana:8.5.0
     ports:
-      - 5601:5601
+      - "5601:5601"
     environment:
       ELASTICSEARCH_URL: http://elasticsearch:9200
       ELASTICSEARCH_HOSTS: '["http://elasticsearch:9200"]'
     networks:
       - argilla
-
 networks:
   argilla:
     driver: bridge
-
 volumes:
-  elasticdata:
+  elasticdata85:
```

Comment:
Leaving the default volume requires an internal elasticsearch migration (from 7.10 to 8.5 cannot be done automatically). 

Changing to a new volume will hide all existing data.

We need to decide on the final option and document it

Reply:
cc @dvsrepo @davidberenstein1957